tests/unit_test/action/test_store_page_action.py

```
@@ -0,0 +1,191 @@
+                   return_value="enc_P-001") as mock_enc, \
+             patch("kairon.actions.definitions.catalog.Authentication.create_access_token",
+                   return_value="tok_abc") as mock_tok, \
+             patch("kairon.shared.actions.data_objects.ActionServerLogs.save") as mock_log:
+            result = await ActionStorePage("bot_cat_exec", "exec_ok").execute(
+                dispatcher, tracker, {}, action_call={}
+            )
+
+        mock_enc.assert_called_once_with("P-001")
+        mock_tok.assert_called_once()
+        assert result["user_identifier"] == "enc_P-001"
+        assert result["temp_token"] == "tok_abc"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "temp_token"

(S105)